{
  "id": "GHSA-77fq-4xf5-hph4",
  "summary": "Buffer overflow in wasm3",
  "details": "WASM3 v0.5.0 was discovered to contain a heap overflow via the component /wabt/bin/poc.wasm.",
  "aliases": [
    "CVE-2022-28990"
  ],
  "modified": "2023-03-07T05:22:03.589985Z",
  "published": "2022-05-21T00:00:26Z",
  "database_specific": {
    "cwe_ids": [
      "CWE-787"
    ],
    "severity": "HIGH",
    "github_reviewed": true,
    "github_reviewed_at": "2022-06-02T14:52:30Z",
    "nvd_published_at": "2022-05-20T19:15:00Z"
  },
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2022-28990"
    },
    {
      "type": "WEB",
      "url": "https://github.com/wasm3/wasm3/issues/323"
    },
    {
      "type": "WEB",
      "url": "https://github.com/wasm3/wasm3/pull/324"
    },
    {
      "type": "WEB",
      "url": "https://github.com/wasm3/wasm3/commit/8441c82c4a98f554b12971450220b580c3c2b6df"
    },
    {
      "type": "PACKAGE",
      "url": "https://github.com/wasm3/wasm3"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "pywasm3",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/pywasm3"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "last_affected": "0.5.0"
            }
          ]
        }
      ],
      "versions": [
        "0.0.1",
        "0.0.2",
        "0.4.8",
        "0.4.9",
        "0.5.0"
      ],
      "database_specific": {
        "source": "https://github.com/github/advisory-database/blob/main/advisories/github-reviewed/2022/05/GHSA-77fq-4xf5-hph4/GHSA-77fq-4xf5-hph4.json"
      }
    }
  ],
  "schema_version": "1.3.0",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
    }
  ]
}